pvlib/irradiance.py

```diff
@@ -515,7 +550,7 @@ def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
         (e.g. surface facing up = 0, surface facing horizon = 90).
 
     ghi : numeric
-        Global horizontal irradiance in W/m^2.
+        Global horizontal irradiance. [W/m^2]
```

Comment:
nit: we should decide on a convention, is it W/m2 or W/m^2? I can't decide. Let's put it to the group.